.github/workflows/build-and-publish-documentation.yaml

```diff
@@ -55,34 +55,10 @@ jobs:
           rmarkdown::render("writeup/si.Rmd", output_dir = "docs")
         shell: Rscript {0}
 
-      - name: Upload documentation
-        uses: actions/upload-artifact@v2
+      - name: Deploy to GitHub pages 🚀
```

Comment:
This is obviously much cleaner but the bit it loses is the ability to inspect docs etc generated by branches other than main which is quite nice when making edits. Is there a way to do that with this action?

Reply:
Oh right, missed this. As it is, it would redeploy on pushes to any branch, which is probably not what you want.

I would test things locally. I'm not sure there is a huge benefit to do it via GitHub Actions. Am I missing something?

Reply:
Mostly that I don't want to test things locally 😆 

Also very handy for collabs etc where they may not have the dev environment and/or may not post what the updated text looks like.